requirements.txt

```diff
@@ -1,12 +1,7 @@
 SQLAlchemy==1.3.10
 geoalchemy2
 psycopg2
-nose2==0.9.1
-testing.postgresql
-black
+git+https://github.com/tk0miya/testing.postgresql.git#egg=testing.postgresql
```

Comment:
@robintw - if this is for running the unit tests, I guess it should be in `requirements_dev.txt`, right?

Reply:
Oh yes, good point! I'll fix that.